transformer_engine/pytorch/module/base.py

```diff
@@ -542,7 +542,7 @@ def prepare_forward(
         inp: torch.Tensor,
         is_first_microbatch: Union[bool, None],
         num_gemms: int = 1,
-    ) -> None:
+    ) -> Generator[torch.Tensor, Any, None]:
```

Comment:
We don't intend to use `send` with this generator:
```suggestion
    ) -> Generator[torch.Tensor, None, None]:
```